CMakeLists.txt

```diff
@@ -131,7 +131,9 @@ else (APPLE)
 endif(APPLE)
 
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
+if (NOT CMAKE_SKIP_RPATH) # Option to not set rpath - required for packaging on Debian
```

Comment:
My understanding is that this is not necessary. If CMAKE_SKIP_RPATH is TRUE then all other RPATH option are ignored.